.github/workflows/chrometest_run-batches.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           C:\Users\runneradmin\Documents\PowerShell\Modules\SimplySql\${{ env.SIMPLY_SQL_VERSION }}
         key: opensans-${{ env.OPENSANS_VERSION }}-fontawesome-${{ env.FONT_AWESOME_VERSION }}-simplysql-${{ env.SIMPLY_SQL_VERSION }}
     - name: Install Dependencies
-      if: steps.cache-dependencies.outputs.cache-hit != 'true'
```

Comment:
Need to just add this check and the install to next job. Fail occurs on "Get Dependencies From Cache" step, whilst this one actually passes. Alternatively, add a repeat of the failing step: https://github.com/Displayr/gha-templates/blob/61dc7901d788582580caaf79636e8e19b72d875e/.github/workflows/chrometest_run-batches.yml#L100-L108.